shared/team-building/user-result.tsx

```diff
@@ -111,11 +99,11 @@ const UserResult = (props: Props) => {
   )
 }
 
-const AvatarSize = 48
+const avatarSize = 48
 const dotSeparator = '•'
 
 const isPreExistingTeamMemberText = (prettyName: string) =>
-  `${prettyName ? prettyName + ` ${dotSeparator} ` : ''} Already in team`
+  `${prettyName ? prettyName + ` ${dotSeparator} ` : ''}Already in team`
```

Comment:
:sheep: you could check for `prettyName === username` here too. Small effect but otherwise I don't think the `!prettyName` case happens currently